content/en/apps/tutorials/targets.md

```diff
@@ -16,53 +16,223 @@ This tutorial will take you through how to build target widgets.
+| Assessment form  | Total assessments | Total number of assessment reports submitted  | Count | All time | _ | No |
+| Assessment form  | Total assessments this month | Total number of assessment reports submitted this month | Count | This month | 2 | No |
+| Assessment form  | Total population with cough  | Total number of household members with cough  | Count | This month | _ | No |
+| Assessment form  | % Population with cough  | Total number of contacts with cough/Total number of contacts assessed | Percent | This month | _ | No |
+| Assessment form  | Total households with assessments  | Total number of households with at least one submitted assessment form  | Count | This month | 2 | Yes |
+| Assessment form  | % Household with >=2 assessments  | Total number of households with at least two patients assessed/Total number of households | Percent | All time | 60 | No |
+
+
 Create a `targets.js` file (this may have already been created by the `initialise-project-layout` command).
 
 ### 1. Define an All-Time Target Widget
-
-Edit the `targets.js` file to define an all-time widget as shown below:
+This widget counts the total number of assessment reports that have been submitted by the user from the time that they started reporting.
+Edit the `targets.js` file to define the total assessments all-time widget as shown below:
 
 ```javascript
-module.exports = [
   {
     id: 'assessments-all-time',
     type: 'count',
     icon: 'icon-healthcare-assessment',
     goal: -1,
     translation_key: 'targets.assessments.title',
     subtitle_translation_key: 'targets.all_time.subtitle',
-
     appliesTo: 'reports',
     appliesToType: ['assessment'],
     date: 'now'
   }
-];
 ```
 
-### 2. Define a Monthly Target Widget
+### 2. Define the Total Assessments Monthly Target Widget
+This widget counts the total number of assessment reports that have been submitted by the user for this month.
+Edit the `targets.js` file and add another target widget definition object to define the total assessments monthly widget as shown below:
+
+```javascript
+  {
+    id: 'assessments-this-month',
+    type: 'count',
+    icon: 'icon-healthcare-assessment',
+    goal: 2,
+    translation_key: 'targets.assessments.title',
+    subtitle_translation_key: 'targets.this_month.subtitle',
+    appliesTo: 'reports',
+    appliesToType: ['assessment'],
+    date: 'reported'
+  }
+```
+
+{{% alert title="Note" %}} All-time widgets have the `date` property set to `now` while monthly widgets have the `date` property set to `reported`. {{% /alert %}}
+
+The images below show the `all-time` and `monthly` target widgets respectively, with the resulting figures varying depending on the number of assessment reports submitted within the respective durations:
+
+![All-time assessements](assessments_all_time_no_translation.png) ![Assessments this month](assessments_this_month_no_translation.png)
```

Comment:
```suggestion
{{< figure src="assessments_all_time_no_translation.png" link="assessments_all_time_no_translation.png" class="right col-6 col-lg-4" >}}
{{< figure src="assessments_this_month_no_translation.png" link="assessments_this_month_no_translation.png" class="right col-6 col-lg-4" >}}

These images show the `all-time` and `monthly` target widgets respectively, with the resulting figures varying depending on the number of assessment reports submitted within the respective durations.
```
Use the 'figure' tag for images. You could also align them to the right as we've been doing based on designer recommendations. Apply the same to the images in the other sections (some are also not showing).